doc/source/deploy/server/ami.rst

```diff
@@ -397,4 +397,141 @@ for the corresponding image:
+--------------------------------------------------------------------------------
+
+The |pmm| data volume is mounted as XFS on top of an LVM volume. There are two
+ways to increase this volume as the |pmm| requirements change.
+
+1. Add a new disk and expand the LVM volume using the new disk.
+2. Expand the existing disk and grow the LVM volume.
+
+There is no significant difference or pros and cons between the two options.
+Option one may have the advantage of growing the volume as the OS permits while
+option two allows to snapshot the single disk from AWS console.
+
+Expanding Single Disk
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To expand the single disk to increase capacity, the following steps can be
+followed.
+
+1. Expand the disk from AWS Console/CLI to the desired capacity.
+2. Login to the |pmm| EC2 instance and verify that the disk capacity has
+   increased. From the ``dmesg`` output, you should see something like below::
+
+     [  535.994494] xvdb: detected capacity change from 17179869184 to 34359738368
+
+3. You can check information about logical volumes and volume groups with the
+   ``lvs`` and ``vgs`` commands::
+
+    [root@ip-10-1-2-70 ~]# vgs
+     VG     #PV #LV #SN Attr   VSize VFree
+     DataVG   1 2 0 wz--n- <16.00g    0
+
+    [root@ip-10-1-2-70 ~]# lvs
+     LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+     DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.74
+     ThinPool DataVG twi-aotz--  15.96g 1.39  1.29|
+
+4. Now we can use the ``lsblk`` command to see that our disk size has been read
+   by the kernel properly but LVM2 does not know about it yet. We can use
+   ``pvresize`` to make sure the PV device reflects the new size. Once
+   ``pvresize`` is executed, we can see that the VG has the new free space
+   available.
+
+   .. code-block:: bash
+
+      [root@ip-10-1-2-70 ~]# lsblk | grep xvdb
+       xvdb                      202:16 0 32G 0 disk
+
+      [root@ip-10-1-2-70 ~]# pvscan
+       PV /dev/xvdb   VG DataVG    lvm2 [<16.00 GiB / 0    free]
+       Total: 1 [<16.00 GiB] / in use: 1 [<16.00 GiB] / in no VG: 0 [0   ]
+
+      [root@ip-10-1-2-70 ~]# pvresize /dev/xvdb
+       Physical volume "/dev/xvdb" changed
+       1 physical volume(s) resized / 0 physical volume(s) not resized|
+
+      [root@ip-10-1-2-70 ~]# pvs
+       PV         VG Fmt Attr PSize   PFree
+       /dev/xvdb  DataVG lvm2 a--  <32.00g 16.00g
+
+5. We can then extend our logical volumes. Since the PMM image uses thin
+   provisioning, we need to extend both the pool and the volume::
+
+      [root@ip-10-1-2-70 ~]# lvs
+       LV       VG Attr       LSize Pool Origin Data%  Meta% Move Log Cpy%Sync Convert
+       DataLV   DataVG Vwi-aotz-- <12.80g ThinPool        1.77
+       ThinPool DataVG twi-aotz--  15.96g 1.42  1.32
+
+      [root@ip-10-1-2-70 ~]# lvextend /dev/mapper/DataVG-ThinPool -l 100%VG
```

Comment:
@fiowro do we always want ThinPool to be 100%?

Reply:
AFAIK it depends on situation. LVM with thin provisioning does not pre-allocate free space on logical volumes. This allows to have pool smaller than logical volumes allocated from it when these logical volumes have enough free (unallocated) space. But perhaps such variants are for some tricky custom mods, and not for the ordinary installation...